### ISSUE TITLE
Cave Biomes do not generate in large modpacks

### ISSUE THREAD

**Issue by enzogfaria** [AlexModGuy/AlexsCaves#141]
Hey! (I'm Brazilian, sorry for any English mistakes). I was having problems with the mod in my modpack since it was released, as it didn't generate the biomes, since then I've been trying to solve this problem by testing with all the mods separately. Hours passed and the only conclusion I could reach is that, if you exceed 365 MODS, the biomes are not generated, exactly 365, if you put 366 it stops working. I tested several different mods and always the same problem when I go above 365. Please help me!

**Comment by Gbergz** [comment]
Can confirm this.

**Comment by melvinczyk** [comment]
Yeah same thing but I only have around 150 mods where 3 only deal with cave gen.

**Comment by Tullemania** [comment]
Having this problem to when passing 365 as well as the mods menu being broken when scrolling past Alex's caves but if you keep scrolling it fixes itself
![Base Profile Screenshot 2023 10 20 - 16 32 42 40](https://github.com/AlexModGuy/AlexsCaves/assets/85764654/aa2d6f03-fab4-4033-aa52-4599c51eb550)
![Base Profile Screenshot 2023 10 20 - 16 32 45 32](https://github.com/AlexModGuy/AlexsCaves/assets/85764654/563e6750-abce-449d-9bff-0f50cf9fdffc)

**Comment by enzogfaria** [comment]
> Yeah same thing but I only have around 150 mods where 3 only deal with cave gen.

What biome mods do you have?

**Comment by melvinczyk** [comment]
> What biome mods do you have?

I only have Biomes O' Plenty, Lucky's Spelunker's Charm II, and Galosphere.

**Comment by wildfire2501** [comment]
i also have biomes o plenty and cannot find any cave biome

**Comment by melvinczyk** [comment]
> i also have biomes o plenty and cannot find any cave biome

No I don't think it is biomes o plenty. I made a separate test modpack with b.o.p in it along with other biome mods and the AC caves still generate so idk.

**Comment by rainysux** [comment]
For me once I pass 358 mods where it says the mod menu, its unable to locate any caves. No matter what I turn off or on

**Comment by knoxhack** [comment]
same here, using Beyond Fantasy: https://legacy.curseforge.com/minecraft/modpacks/beyond-fantasy/files/4809755

**Comment by enzogfaria** [comment]
issue solved with citadel 2.4.6

**Comment by MochiVN634** [comment]
> issue solved with citadel 2.4.6

And where do we download Citadel 2.4.6?

**Comment by AlexModGuy** [comment]
you'll have to wait for curseforge to approve it, likely later today or tomorrow.

**Comment by MochiVN634** [comment]
> you'll have to wait for curseforge to approve it, likely later today or tomorrow.

Tried with both Citadel 2.4.6 and 2.4.7, unfortunately, the biomes still aren't being generated in my game.

**Comment by Gbergz** [comment]
Works for me now with Citadel 2.4.7.
